Explorer/Assets/Scripts/ECS/SceneLifeCycle/SceneDefinition/Systems/LoadSceneDefinitionListSystem.cs

```diff
@@ -32,7 +32,7 @@ public partial class LoadSceneDefinitionListSystem : LoadSystemBase<SceneDefinit
         // There is no cache for the list but a cache per entity that is stored in ECS itself
         internal LoadSceneDefinitionListSystem(World world, IStreamableCache<SceneDefinitions, GetSceneDefinitionList> cache,
             MutexSync mutexSync)
-            : base(world, cache, mutexSync) { }
+            : base(world, cache, mutexSync, new NullBudgetProvider()) { }
```

Comment:
Also if you really want to budget some specific loading systems (and their synchronous logic) it does not look sensible to include `BudgetProvider` in a base class.